### ISSUE TITLE
FemtoVG / Windows: Window content lag when resizing window

### ISSUE THREAD

**Issue by teknalb** [slint-ui/slint#5563]
Slint v1.6.0
Rust v1.79.0

https://github.com/slint-ui/slint/assets/174643755/3efcbefb-5444-4e25-93ba-1ba425a65b3b

**Comment by tronical** [comment]
I can't quite reproduce this, but I guess the way the GL driver handles surface resizing depends also on the driver here.

I'm curious, is this the same if you do a release build?

**Comment by ogoffart** [comment]
is the problem hapenning with both the femtovg and skia renderer?

**Comment by tronical** [comment]
If this also happens with `renderer-skia` as well as `renderer-skia-opengl`, then perhaps this is similar to #5791.

**Comment by tronical** [comment]
Thank you for testing. Interesting that this is specific to FemtoVG then - hard to image why... unless this is "simply" a performance issue. When resizing the window with femtovg, do you see also a spike in CPU usage compared to Skia?

**Comment by tronical** [comment]
True, that's a little higher, but not by that much. I looked at the code I don't failing to find a good explanation here. When resizing with FemtoVG and with Skia OpenGL, we call exactly the same code to resize the surface.

**Comment by lmaxyz** [comment]
> True, that's a little higher, but not by that much. I looked at the code I don't failing to find a good explanation here. When resizing with FemtoVG and with Skia OpenGL, we call exactly the same code to resize the surface.



> True, that's a little higher, but not by that much. I looked at the code I don't failing to find a good explanation here. When resizing with FemtoVG and with Skia OpenGL, we call exactly the same code to resize the surface.



> True, that's a little higher, but not by that much. I looked at the code I don't failing to find a good explanation here. When resizing with FemtoVG and with Skia OpenGL, we call exactly the same code to resize the surface.

 Is it still reproduces if you add some real-time animation (for example permanent boxes background changing)?

**Comment by Enyium** [comment]
@teknalb:

>  renderer-skia -----------> weird subtle scaling issue when resizing

Maybe like in #5765?

**Comment by lmaxyz** [comment]
@teknalb 
 Is it still reproduces if you add some real-time animation (for example permanent boxes background changing)?
